src/timbot.py

```diff
@@ -166,20 +165,12 @@ def main():
     if dev_mode:
         use_authenticated_user = False
 
-    # load configuration data
-    try:
-        with open('config.yaml', 'r') as file:
-            config = yaml.load(file)
-    except Exception as e:
-        print("Error loading configuration data: ", e)
-        sys.exit()
-
     # connect to database
     try:
-        conn = mysql.connector.connect(user=config['mysql']['user'],
-                                       password=config['mysql']['password'],
-                                       host=config['mysql']['host'],
-                                       database=config['mysql']['database'])
+        conn = mysql.connector.connect(user=os.environ['SQL_USER'],
```

Comment:
This is going to produce a key error exception if that environment variable doesn't exist.

Reply:
better set it then

Reply:
even though this wasn't wrapped before, i will wrap it this time in next patchset